vm-setup/roles/v1aX_integration_test/tasks/upgrade.yml

```diff
@@ -474,28 +474,17 @@
       KUBECONFIG: "/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml"
     ignore_errors: yes
 
-  - name: Verify that the old controlplane node has left the cluster
-    shell: |
-            kubectl get bmh -n {{NAMESPACE}} | grep -i provisioned | grep -c "{{ CLUSTER_NAME }}-controlplane-"
-    environment:
-      KUBECONFIG: "/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml"
-    retries: 200
-    delay: 20
-    register: upgraded_cp_nodes_count
-    until: upgraded_cp_nodes_count.stdout|int == 0
-    failed_when: upgraded_cp_nodes_count.stdout|int != 0
-
-  - name: Get control plane machines
```

Comment:
We don't need this check anymore?

Reply:
line 477: We don't need that task as we are using scale-in feature, there is no old control plane in the scenario. 
line:488 I changed the name of that task only.

Reply:
Ah right